tests/ecdhp256-test.c

```diff
@@ -89,8 +101,75 @@ int main()
 	ok = ok && compare_and_print(64, point_compressed, compressed7);
 
 
+// The start of Wycherproof tests
+
+	printf("%s\n", "Wycheproof tests ECDH: ");
+		
+	for (int i = 0 ; i< sizeof(w_vectors)/sizeof(ecdhp256_w_i); i++)
+	{
+		printf("ECDH Wycheproof Test %d \n", i );
+
+		bool flagDecompressSuccessful = Hacl_P256_decompression_not_compressed_form(w_vectors[i].publicKey, decompressedPoint);
```

Comment:
can you remind me why one function returns a `bool` for success (0 == error, 1 == success) and the other one a `uint64_t` (0 == success, MAX_UINT64 == error)? I'm very concerned about C clients getting this wrong